source/iwyu.imp

```diff
@@ -0,0 +1,377 @@
+[
```

Comment:
What does this file do?

Reply:
it maps headers from internal headers to headers that should be used
see https://github.com/include-what-you-use/include-what-you-use/blob/master/docs/IWYUMappings.md